### PR TITLE
COMMERCE-9815

### PR DESCRIPTION
**Pull request by Brunno0liveira**
https://liferay.atlassian.net/browse/COMMERCE-9815

### CODE REVIEW

**Comment by Brunno0liveira** [comment]
ci:test:sf

**Comment by Brunno0liveira** [comment]
ci:test:relevant

File: modules/apps/commerce/commerce-product-test/src/testFunctional/tests/CPCommerceProductOptions.testcase [hunk 1]
```diff
@@ -262,9 +260,21 @@ definition {
...
+				key_toggleSwitchLabel = "Default",
+				locator1 = "ToggleSwitch#ANY_TOGGLE_SWITCH");
+
+			Button.clickSave();
 
 			Alert.viewSuccessMessage();
+
+			SelectFrameTop();
+
+			Click(locator1 = "Icon#CLOSE");
```
Comment (pietrobocale) [inline modules/apps/commerce/commerce-product-test/src/testFunctional/tests/CPCommerceProductOptions.testcase:277]: After saving, please add an assert to verify if the toggle has successfully changed the status.

File: modules/apps/commerce/commerce-product-test/src/testFunctional/tests/CPCommerceProductOptions.testcase [hunk 2]
```diff
@@ -327,9 +335,21 @@ definition {
...
+				checkboxName = "Default",
+				locator1 = "Checkbox#ANY_CHECKBOX");
+
+			Button.clickSave();
 
 			Alert.viewSuccessMessage();
+
+			SelectFrameTop();
+
+			Click(locator1 = "Icon#CLOSE");
```
Comment (pietrobocale) [inline modules/apps/commerce/commerce-product-test/src/testFunctional/tests/CPCommerceProductOptions.testcase:352]: After saving, please add an assert to verify if the toggle has successfully changed the status.

**Review by pietrobocale — CHANGES_REQUESTED** [review]
Hey @Brunno0liveira, I asked for some changes. Please take a look at the comments.

Thanks!

**Comment by pietrobocale** [comment]
Note that we are changing an existing test. The old one used the Toggle Default option to enable and disable the Default. Now, with these changes, we are changing the scope of the test, so we are removing coverage.

So @Brunno0liveira, if you want to change the old test as you did, you need to add a new test that covers the removed scenario.
Or the other solution is to remain the old test like it was and add a new one that covers this new scenario.

Thanks!

**Comment by Brunno0liveira** [comment]
ci:test:sf

**Comment by Brunno0liveira** [comment]
ci:test:relevant

**Review by pietrobocale — APPROVED** [review]
Review approved just change the name in  `CanAssertDefaultOptionValuesAreSavedUsingToggleSwitch` and is ready to go. Please check the CI to ensure that the tests are passing.

Thanks!

**Comment by Brunno0liveira** [comment]
ci:test:sf

**Comment by Brunno0liveira** [comment]
ci:test:relevant

**Comment by Brunno0liveira** [comment]
ci:test:commerce-functional

**Comment by Brunno0liveira** [comment]
ci:test:relevant

**Comment by Brunno0liveira** [comment]
ci:test:relevant

**Comment by Brunno0liveira** [comment]
The issues are not related to the changes, PR can be forward, thanks!

@lorenzolapescara

**Comment by andrea-ale-sbarra** [comment]
ci:forward:force

**Comment by lorenzolapescara** [comment]
ci:test:relevant

**Comment by victoroscuro** [comment]
ci:forward:force

**Pull request closed without merging**